sycl/doc/UsersManual.md

```diff
@@ -410,6 +410,13 @@ and not recommended to use in production environment.
     Also, some of Intel GPUs or GPU run-time/drivers may support only
     "stateless" memory accesses.
 
+**`-f[no-]target-export-symbols`**
+
+    Exposes exported symbols in a generated target library to allow for
+    visibility to other modules.
+
+    NOTE: This flag is only supported for spir64_gen AOT targets.
```

Comment:
What is the expected behavior if this flag is used for targets other than `spir64_gen` ?

Reply:
The option will effectively be a no-op.  Under non AOT spir64_gen conditions, the driver will emit a warning that the option is unused.

Reply:
Maybe we could consider adding a test case to capture this case.